### PR TITLE
Just another cleanup

### PR DESCRIPTION
**Pull request by amazingalek**

### CODE REVIEW

File: OWML.ModLoader/ModData.cs [hunk 1]
```diff
@@ -77,17 +79,15 @@ private void MakeConfigConsistentWithDefault()
...
             var userString = userSetting is JObject objectValue ? (string)objectValue["value"] : Convert.ToString(userSetting);
-            Config.Settings[key] = modderSetting;
-            var isInOptions = options.Contains(userString);
-            if (isInOptions)
+            Config.Settings[key] = modSetting;
+            if (options.Contains(userString))
             {
                 Config.SetSettingsValue(key, userString);
             }
-            return isInOptions;
```
Comment (TAImatem) [inline OWML.ModLoader/ModData.cs:90]: nope, that's actually important (wait for new PR a bit)

File: OWML.ModLoader/ModData.cs [hunk 2]
```diff
@@ -96,41 +96,39 @@ private void AddMissingDefaults(IModConfig defaultConfig)
             missingSettings.ForEach(setting => Config.Settings.Add(setting.Key, setting.Value));
         }
 
-        private bool TryUpdate(string key, object userSetting, object modderSetting)
+        private void TryUpdate(string key, object userSetting, object modSetting)
```
Comment (TAImatem) [inline OWML.ModLoader/ModData.cs:99]: same

File: OWML.ModLoader/ModData.cs (hunk 1 above)
Comment (amazingalek) [reply OWML.ModLoader/ModData.cs]: reverted

File: OWML.ModLoader/ModData.cs (hunk 2 above)
Comment (amazingalek) [reply OWML.ModLoader/ModData.cs]: reverted

File: OWML.ModHelper.Menus/ModInputCombinationPopup.cs [hunk 3]
```diff
@@ -89,7 +89,7 @@ protected override void Update()
                 return;
             }
             if (currentlyPressedKeys.Count < 8 && (currentlyPressedKeys.Count > _combination.Count
-                || (currentlyPressedKeys.Count > 1 && _wasReleased)))
+                || currentlyPressedKeys.Count > 1 && _wasReleased))
```
Comment (TAImatem) [inline OWML.ModHelper.Menus/ModInputCombinationPopup.cs:92]: I wouldn't be so brave with bools in your place 🤔

Comment (TAImatem) [reply OWML.ModHelper.Menus/ModInputCombinationPopup.cs:92]: in this case it seems to be fine, but in some languages some boolean operators have same precedence tier

File: OWML.ModLoader/ModData.cs [hunk 4]
```diff
@@ -121,16 +123,16 @@ private bool TryUpdate(string key, object userSetting, object modderSetting)
...
         private bool IsNumber(object setting)
         {
-            if (setting is JObject settingObject)
-            {
-                return settingObject["type"].ToString() == "slider";
-            }
-            return new[] { typeof(long), typeof(int), typeof(float), typeof(double) }.Contains(setting.GetType());
+            return setting is JObject settingObject
+                ? settingObject["type"].ToString() == "slider"
+                : new[] { typeof(long), typeof(int), typeof(float), typeof(double) }.Contains(setting.GetType());
```
Comment (TAImatem) [inline OWML.ModLoader/ModData.cs:128]: thought you would do that 😅

**Pull request merged**